Listing '.'...
Listing './bin'...
Compiling './bin/wsdump.py'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/certifi'...
Listing './lib/python3.5/site-packages/certifi-2017.11.5.dist-info'...
Listing './lib/python3.5/site-packages/chardet'...
Listing './lib/python3.5/site-packages/chardet/cli'...
Listing './lib/python3.5/site-packages/chardet-3.0.4.dist-info'...
Listing './lib/python3.5/site-packages/idna'...
Listing './lib/python3.5/site-packages/idna-2.6.dist-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pypyr'...
Compiling './lib/python3.5/site-packages/pypyr/cli.py'...
***   File "./lib/python3.5/site-packages/pypyr/cli.py", line 39
    version=f'{pypyr.version.get_version()}')
                                           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/context.py'...
***   File "./lib/python3.5/site-packages/pypyr/context.py", line 34
    raise KeyNotInContextError(f"{key} not found in the pypyr context.")
                                                                      ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyr/log'...
Compiling './lib/python3.5/site-packages/pypyr/log/logger.py'...
***   File "./lib/python3.5/site-packages/pypyr/log/logger.py", line 35
    f"Root logger {root_logger.name} configured with level "
                                                           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/moduleloader.py'...
***   File "./lib/python3.5/site-packages/pypyr/moduleloader.py", line 30
    logger.debug(f"loading module {module_abs_import}")
                                                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyr/parser'...
Compiling './lib/python3.5/site-packages/pypyr/parser/jsonfile.py'...
***   File "./lib/python3.5/site-packages/pypyr/parser/jsonfile.py", line 17
    logger.debug(f"attempting to open file: {context_arg}")
                                                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/parser/yamlfile.py'...
***   File "./lib/python3.5/site-packages/pypyr/parser/yamlfile.py", line 17
    logger.debug(f"attempting to open file: {context_arg}")
                                                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/pipelinerunner.py'...
***   File "./lib/python3.5/site-packages/pypyr/pipelinerunner.py", line 37
    logger.debug(f"context parser found: {parser_module_name}")
                                                             ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyr/pipelines'...
Listing './lib/python3.5/site-packages/pypyr/steps'...
Compiling './lib/python3.5/site-packages/pypyr/steps/assert.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/assert.py", line 31
    assert context, f"context must have value for {__name__}"
                                                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/contextclear.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/contextclear.py", line 40
    logger.debug(f"removing {k} from context")
                                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/contextclearall.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/contextclearall.py", line 19
    logger.info(f"Context wiped. New context size: {len(context)}")
                                                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/contextset.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/contextset.py", line 42
    logger.debug(f"setting context {k} to value from context {v}")
                                                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/contextsetf.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/contextsetf.py", line 39
    logger.debug(f"setting context {k} to value from context {v}")
                                                                ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/env.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/env.py", line 24
    assert context, f"context must have value for {__name__}"
                                                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/fetchjson.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/fetchjson.py", line 38
    logger.debug(f"attempting to open file: {file_path}")
                                                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/fetchyaml.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/fetchyaml.py", line 37
    logger.debug(f"attempting to open file: {file_path}")
                                                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/fileformat.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/fileformat.py", line 36
    logger.debug(f"opening source file: {in_path}")
                                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/fileformatjson.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/fileformatjson.py", line 41
    logger.debug(f"opening json source file: {in_path}")
                                                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/fileformatyaml.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/fileformatyaml.py", line 41
    logger.debug(f"opening yaml source file: {in_path}")
                                                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/filereplace.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/filereplace.py", line 53
    logger.debug(f"opening source file: {in_path}")
                                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/py.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/py.py", line 25
    logger.debug(f"Executing python string: {context['pycode']}")
                                                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/pype.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/pype.py", line 56
    logger.info(f"pyping {pipeline_name}, using parent context.")
                                                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/pypyrversion.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/pypyrversion.py", line 13
    logger.info(f"pypyr version is: {pypyr.version.get_version()}")
                                                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/safeshell.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/safeshell.py", line 36
    logger.debug(f"Processing command string: {context['cmd']}")
                                                              ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/shell.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/shell.py", line 42
    logger.debug(f"Processing command string: {context['cmd']}")
                                                              ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/steps/tar.py'...
***   File "./lib/python3.5/site-packages/pypyr/steps/tar.py", line 39
    assert context, f"context must have value for {__name__}"
                                                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/stepsrunner.py'...
***   File "./lib/python3.5/site-packages/pypyr/stepsrunner.py", line 23
    logger.debug(f"retrieving {steps_group} steps from pipeline")
                                                               ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyr/utils'...
Compiling './lib/python3.5/site-packages/pypyr/utils/poll.py'...
***   File "./lib/python3.5/site-packages/pypyr/utils/poll.py", line 29
    logger.debug(f"Looping every {interval} seconds for "
                                                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyr/version.py'...
***   File "./lib/python3.5/site-packages/pypyr/version.py", line 9
    return (f'pypyr {__version__} '
                                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyr-0.6.1.dist-info'...
Listing './lib/python3.5/site-packages/pypyrslack'...
Listing './lib/python3.5/site-packages/pypyrslack/steps'...
Compiling './lib/python3.5/site-packages/pypyrslack/steps/send.py'...
***   File "./lib/python3.5/site-packages/pypyrslack/steps/send.py", line 43
    f"Sending to {context['slackChannel']}: {context['slackText']}")
                                                                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pypyrslack/version.py'...
***   File "./lib/python3.5/site-packages/pypyrslack/version.py", line 10
    return (f'pypyrslack {__version__} '
                                       ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pypyrslack-0.3.0.dist-info'...
Listing './lib/python3.5/site-packages/requests'...
Listing './lib/python3.5/site-packages/requests-2.18.4.dist-info'...
Listing './lib/python3.5/site-packages/ruamel'...
Listing './lib/python3.5/site-packages/ruamel/yaml'...
Listing './lib/python3.5/site-packages/ruamel.yaml-0.14.12.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/six-1.11.0.dist-info'...
Listing './lib/python3.5/site-packages/slackclient'...
Listing './lib/python3.5/site-packages/slackclient-1.1.0.egg-info'...
Listing './lib/python3.5/site-packages/urllib3'...
Listing './lib/python3.5/site-packages/urllib3/contrib'...
Listing './lib/python3.5/site-packages/urllib3/contrib/_securetransport'...
Listing './lib/python3.5/site-packages/urllib3/packages'...
Listing './lib/python3.5/site-packages/urllib3/packages/backports'...
Listing './lib/python3.5/site-packages/urllib3/packages/ssl_match_hostname'...
Listing './lib/python3.5/site-packages/urllib3/util'...
Listing './lib/python3.5/site-packages/urllib3-1.22.dist-info'...
Listing './lib/python3.5/site-packages/websocket'...
Listing './lib/python3.5/site-packages/websocket/tests'...
Listing './lib/python3.5/site-packages/websocket/tests/data'...
Listing './lib/python3.5/site-packages/websocket_client-0.46.0.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
